packages/Activity-Tracker-browser-extension/manifest.json

```diff
@@ -0,0 +1,29 @@
+{
+    "name":"Browser Activity Tracker",
+    "description": "Shows you your digital time based on your browser activity",
+    "version": "1.0.0",
+    "manifest_version": 3,
+    
+    "action":{
+        "default_icon":{
+            "16": "images/icon16.png",
+            "32": "images/icon32.png",
+            "48": "images/icon48.png",
+            "128": "images/icon128.png"
+        }
+        
+    },
+
+    "background":{
+        "service_worker": "src/background.js"
+    },
+
+    "author": "",
+    "homepage_url": "https://github.com/OpenLake/Activity-Tracker#readme",
```

Comment:
Because in `Activity-Tracker/package.json` on [line 28](https://github.com/OpenLake/Activity-Tracker/blob/e092504f40ae6353ce13e336d172ef08a7d3c59c/package.json#L28) it was written like this.

So I just kept it as it was